std/haxe/iterators/HashMapKeyValueIterator.hx

```diff
@@ -0,0 +1,28 @@
+package haxe.iterators;
+
+import haxe.ds.HashMap;
+
+class HashMapKeyValueIterator<K:{function hashCode():Int;}, V> {
```

Comment:
This is almost an exact copy of `MapKeyValueIterator`, but still needed, I think.